pyvrp/cpp/search/OptionalInsert.h

```diff
@@ -0,0 +1,34 @@
+#ifndef PYVRP_SEARCH_OPTIONAL_INSERT_H
+#define PYVRP_SEARCH_OPTIONAL_INSERT_H
+
+#include "PerturbationOperator.h"
+#include "ProblemData.h"
+
+namespace pyvrp::search
+{
+/**
+ * Optional insert perturbation operator. This operator forcefully inserts
+ * optional clients into the solution.
+ */
+class OptionalInsert : public PerturbationOperator
+{
+    ProblemData const &data_;
+
+public:
+    /**
+     * Creates an optional insert operator.
+     *
+     * Parameters
+     * ----------
+     * data
+     *     Problem data instance.
+     */
```

Comment:
This needs to be in the class-level docstring, if necessary. We do not document constructors like this (because it's never used in the bindings, only the class-level docstring is).

Class-level docstring also needs a first line type hint,
```
 * OptionalInsert(data: ProblemData)
```
should do

Reply:
Oh, I missed this with the other operators. Could you also update those?